### PR TITLE
Console-3249: Trim whitespace from when creating image pull Secret

### PR DESCRIPTION
**Pull request by DcooperRH**
https://issues.redhat.com/browse/CONSOLE-3249

**Changes to _public/components/secrets/create-secret.tsx_:** 
Added an onBlur event handler to trim whitespace to the form inputs - _Registry Server Address, Username, Password_ and _Email_ when creating/editing an image pull secret.

**File addition - _packages/integration-tests-cypress/tests/crud/image-pull-secret.spec.ts_:** 
This integration test creates an image pull secret and adds additional whitespace to the values on the inputs _Registry Server Address, Username, Password_ and _Email_. 
After adding these values and before saving the secret, the test checks that the input fields automatically removed the whitespace.

### CODE REVIEW

File: frontend/packages/integration-tests-cypress/tests/crud/image-pull-secret.spec.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    .get('body')
+    .then(($body) => {
+      if ($body.find(`[data-test-dropdown-menu="image"]`).length) {
+        cy.get(`[data-test-dropdown-menu="image"]`).click();
+      }
+    });
+};
+
+const typeValue = (testId: string, inputValue: string, blur: boolean) => {
+  !blur
```
Comment (jhadvig) [inline frontend/packages/integration-tests-cypress/tests/crud/image-pull-secret.spec.ts:17]: Do we need to call the `blur()` ?
Wont the focus get lost, once next field will get focus to enter the value?

**Review by jhadvig — APPROVED** [review]
/approve
/lgtm

**Comment by yapei** [comment]
@yanpzhan please help test, thanks!

**Comment by yanpzhan** [comment]
@DcooperRH Hi, tested against the pr, after create image pull secret with whitespace in fields Registry Server Address, Username, Password and Email. Check on secret details page, click "Reveal values", the whitespace in email address are still there. See screenshot:

**Comment by yanpzhan** [comment]
@DcooperRH Have tested the pr again, the issue about whitespace in mail address has been resloved. Could you help to review test case for the feature? I've pasted [case link ]( https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-54213)in jira CONSOLE-3249, if the case is accepted, pls help to add tc-approved label in jira, thanks!

**Review by jhadvig — APPROVED** [review]
/lgtm
/hold cancel
